### ISSUE TITLE
How is your accuracy result

### ISSUE THREAD

**Issue by taosiqin1991** [jiegzhan/multi-class-text-classification-cnn-rnn#18]
I want to learn about how  the best accuracy of your methods is and the  comparisons of all your methods.

**Comment by ansipes** [comment]
With the supplied test, the model achieves ~99% accuracy. Keep in mind the training set is very large compared to the test set.
